content/blog/2020-08-27-august-20-community-gems.md

```diff
@@ -96,7 +96,7 @@ Alternately, if your pipeline stage has already been created, you can manually
 edit your `dvc.yaml` file to replace `params.yaml` with `params.json`.
 
 For more about the `params.yaml` file,
-[see our docs](https://dvc.org/doc/start/experiments#defining-parameters).
+[see our docs](https://dvc.org/doc/start/metrics-parameters-plots#defining-parameters).
```

Comment:
Nice. There´s another link in content/blog/2020-12-18-december-20-dvc-heartbeat.md

Reply:
Oh I see you already noticed this one.

> reference in https://dvc.org/blog/december-20-dvc-heartbeat to the old video. That was removed from the page since it is more focused on experiments. Thoughts on how to handle this reference, but also what to do with that video?

I guess in this case given the context we should leave the link as-is. I guess we'll try to make new videos for these 2 pages in the future?